tests/ocpp-server/README.md

```diff
@@ -1,23 +1,33 @@
 # OCPP2 Mock Server
 
 This project includes an Open Charge Point Protocol (OCPP) version 2.0.1 mock server implemented in Python.
+https://github.com/mobilityhouse/ocpp/blob/master/ocpp/v201/enums.py
 
 ## Prerequisites
 
 This project requires Python 3.7+ and [poetry](https://python-poetry.org/) to install the required packages:
+```shell
+pip install poetry
+```
+then
```

Comment:
In my local environment, I needed to use "pip install poetry".

Ok, I'm going to add the likn.

Reply:
Since you seems to develop under an Unix like OS, you can use homebrew on OS X or the package manager of your Unix derivative OS to install and maintain all the development stack needed without any supplemental effort. 

And on Unix, /etc in the file hierarchy standard is meant for configuration, not for third party binaries :) /opt/<name> or /usr/local are the standardized paths for binaries manual installation. 

As long as your current installation works without creating a venv in the cloned repo ...